.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+######Fixes #[ISSUE]
+
+####Changes proposed:
+* [ ] Add
+* [ ] Prepare
+* [ ] Delete
+* [ ] Fix
+
+###Things I've done:
+* [ ] I've pulled the latest master branch
+* [ ] I've run `npm install` to install all the dependenies
+* [ ] I've checked my code, it is clean. (I used jsDoc.)
```

Comment:
No, at the moment we're using a custom build process. Eslint validations are not automatically applied when building. Everyone should be observing those rules at code time by using the Eslint Extension for VSCode and downloading the npm packages already referred in the project's package json.